src/app/pages/DashboardPage/DurationSelect/index.tsx

```diff
@@ -17,7 +16,7 @@ interface DurationSelectProps {
 const DurationSelectCmp: FC<DurationSelectProps> = ({ defaultValue = ChartDuration.TODAY, handleChange }) => {
   const { t } = useTranslation()
 
-  const options = useConstant<DurationOption[]>(() => [
```

Comment:
`useConstant` is actually a performance optimization. As it uses ref behind the scenes. The current implementation will create a new instance of `options` every time rerender happens. To prevent this from happening you can define the constant outside of the Component.

Reply:
I would say it's a premature optimization, and in this case, if we had a way to switch the language, it would break translations